spec/takeaway_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let (:item){ double :item }
+
+  it 'should have a function to view the menu' do
+    takeaway = Takeaway.new
+    expect(takeaway).to respond_to(:menu)
+  end
+
+  it 'should show you the menu when you call the function' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu).to be_an_instance_of(Hash)
+  end
+
+  it 'should list a number of items and prices' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu).not_to be_empty
+  end
+
+  it 'should show a number of items' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu.length).to be > 5
```

Comment:
This test is tying you to a specific implementation. Future clients might want to have a shorter menu than this.
You could also make your code more reusable by not hard-coding the menu: https://github.com/nboyse/takeaway-challenge/blob/master/docs/review.md#open-closed-principle